demo/start.sh

```diff
@@ -85,6 +85,7 @@ for port in 8080 8081 8082; do
             echo 'trusted_key_servers:'
```

Comment:
While we're here, I'm not sure what the "Ignore keys from the trusted keys server" comment above is talking about. Did it mean to say "Ignore keys from the other demo synapses"?